infrastructure/smart-contracts/contracts/RefereeCalculations.sol

```diff
@@ -32,6 +32,8 @@ contract RefereeCalculations is Initializable, AccessControlUpgradeable {
      *
      * @param totalSupply The current total supply of tokens.
      * @param maxSupply The maximum supply of tokens.
+     * @param challengeStart The timestamp of the last challenge submitted.
+     * @param challengeEnd  The current timestamp for this challenge.
```

Comment:
I think the variable names in that function are not accurate since the calcualtion of the reward based on a timespan logically has nothing to do with a challenge. 

Feel free to upadte that comment in this PR or whenever the next person stumbles accross this. I don't think its part of the audit fix, as this was a solidity compile warning not having all params in the function doc.

Let me know what you think, or feel free to request changes if you believe this should be updated here.